lib/razor/config.rb

```diff
@@ -54,19 +54,23 @@ def initialize(env, fname = nil, defaults_file = nil)
               _("The configuration defaults file %{filename} is not readable") % {filename: defaults_file}
       end
 
-      begin
-        yaml = File::open(fname, "r") { |fp| YAML::load(fp) } || {}
-
-        @values.merge!(yaml["all"] || {})
-        @values.merge!(yaml[Razor.env] || {})
-      rescue Errno::ENOENT
-        # This is only an error if we don't have any config values yet.
-        raise InvalidConfigurationError,
-          _("The configuration file %{filename} does not exist") % {filename: fname} if @values.empty?
-      rescue Errno::EACCES
-        raise InvalidConfigurationError,
-          _("The configuration file %{filename} is not readable") % {filename: fname}
+      if File.readable?(fname)
```

Comment:
Main change in https://github.com/puppetlabs/pe-razor-server/commit/bb315d013b113dc480bc101e89b060abfc6e7115 for this file.